packages/web/src/pages/chat-page/components/CreateChatUserResult.tsx

```diff
@@ -109,6 +109,7 @@ export const MessageUserSearchResult = (props: UserResultComposeProps) => {
     if (canCreateChat) {
       closeParentModal()
       dispatch(createChat({ userIds: [user.user_id] }))
+      closeParentModal()
```

Comment:
this seems odd. Do we need this bc there are 2 layers of parent modals?

Reply:
nah, bad merge, 110 takes care of this